docs/1.0. EDID-Mapping.md

```diff
@@ -0,0 +1,195 @@
+# AMWA BCP-XXX-YY: EDID Mapping in NMOS Systems \[Work In Progress\]
+{:.no_toc}
+
+* A markdown unordered list which will be replaced with the ToC, excluding the "Contents header" from above
+{:toc}
```

Comment:
This does not render with the GitHub UI 🙊 

https://github.com/AMWA-TV/nmos-edid-connection-management/blob/805157468dd8b14d28edb833a5faaf1090bc35d2/docs/1.0.%20EDID-Mapping.md